### DISCUSSION TITLE
Ability to delete an event

### DISCUSSION THREAD

**Discussion by aureliusm** [saade/filament-fullcalendar#37] [Ideas]
Hi,

would it be possible to add a DELETE button on the right side in the edit modal?

It is impossible to delete an event right now.

Thanks.

**Comment by saade** [comment]
Sure

**Reply by aureliusm** [reply]
@saade Do you maybe have a timeline for this feature? Thanks :)

**Reply by saade** [reply]
I dont have any timeline for this, i can have a look some day in the next weeks, but if you could PR that, it will be easier for me :)
